house/forms.py

```diff
@@ -0,0 +1,23 @@
+
+from django import forms
+
+from house.models import House, Job
+
+
+class HouseForm(forms.ModelForm):
+    class Meta:
+        model = House
+        fields = ('country', 'city', 'children')
+
+    jobs = Job.choices.copy()
+    jobs.append((None, '---------'))
+    parent_profession_1 = forms.ChoiceField(choices=jobs, required=False)
+    parent_profession_2 = forms.ChoiceField(choices=jobs, required=False)
+    highest_income = forms.IntegerField(label='highest income', required=False)
+    lowest_income = forms.IntegerField(label='lowest income', required=False)
+
```

Comment:
I would change the income to be shown like that:
![image](https://user-images.githubusercontent.com/57755873/164175528-434ca914-4652-4d77-93ee-71cb693b47c7.png)

Reply:
Sure, add it as an issue.

Reply:
Sure I will add